src/components/toolbars/content-toolbar.js

```diff
@@ -131,22 +131,32 @@ const ContentToolbar = (props) => {
         [selectedElements, userId]
     );
 
-    const allowsDelete = useMemo(() => isUserAllowed, [isUserAllowed]);
+    const noCreationInProgress = useMemo(
+        () => selectedElements.every((el) => el.hasMetadata),
+        [selectedElements]
+    );
+
+    const allowsDelete = useMemo(
```

Comment:
i think allowsDelete is not particularly expensive, it use an already memoized value.